.github/workflows/next-build.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: build-next
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    -
+      name: "Set up QEMU"
+      uses: docker/setup-qemu-action@v1
+    -
+      name: "Set up Docker Buildx"
+      uses: docker/setup-buildx-action@v1
+    -
+      name: "Docker quay.io Login"
+      uses: docker/login-action@v1
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+    - 
+      name: Clone source code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - 
+      name: Prepare
+      id: prep
+      run: |
+        set -e
+        SHORT_SHA1=$(git rev-parse --short HEAD)
+        echo ::set-output name=short_sha1::${SHORT_SHA1}
+        echo ::set-output name=version::next
+        IMAGE=kubernetes-image-puller
+        echo ::set-output name=image::${IMAGE}
+    -
+      name: "Build and push"
```

Comment:
don't you need to run make at some point? https://github.com/che-incubator/kubernetes-image-puller#building

Reply:
you mean https://github.com/che-incubator/kubernetes-image-puller/blob/main/docker/Dockerfile#L39 ?